src/platforms/px4_module.h

```diff
@@ -114,7 +115,7 @@ template<class T>
 class ModuleBase
 {
 public:
-	ModuleBase() = default;
+	ModuleBase() : _task_should_exit{false} {}
```

Comment:
No longer default?

Reply:
This is due to a compiler bug: https://github.com/PX4/Firmware/pull/11328/commits/adca15520ae835477d80e1c4da2afbe7e76edf1c.
Ideally we should update these toolchains.